src/handbook/peopleops/hiring.md

```diff
@@ -154,3 +154,55 @@ for the open position, take the job posting offline. Candidates applying for a j
 that's no longer available is a waste of their time and a bad experience with
 FlowForge and hurts our reputation.
 
+## Offboarding
+The purpose of the offboarding section in the handbook is to provide a clear and structured framework for employees at FlowForge to follow during the offboarding process. The offboarding process is managed by the People Ops Manager in close collaboration with the CEO, Reporting Managers, and Deel for Payroll at FlowForge.
+
+When a team member or contractor decides to leave FlowForge, they will receive a detailed email before their final date of employment. The email will include information regarding the impact on benefits coverage, stocks, final pay, and the process for returning any company assets.
+
+### Voluntary Resignation
+
+If a team member decides to leave FlowForge, we request them to provide a 30-day notice period by default, unless specified otherwise in their employment contract on Deel. During this period, we expect the team member to continue working to facilitate a smooth handover and enable us to find a suitable replacement. Please note that we generally expect the team member to work for the entire notice period.
```

Comment:
Let's keep the contracts the contracts, not in the handbook to avoid confusion.

```suggestion
If a team member decides to leave FlowForge they're expected to continue working to facilitate a smooth handover and enable us to find a suitable replacement. Please note that we generally expect the team member to work for the entire notice period.
```